src/main/kotlin/io/github/mojira/arisa/modules/CommandModule.kt

```diff
@@ -30,15 +32,15 @@ class CommandModule(
             assertNotEmpty(staffComments).bind()
 
             val results = staffComments
-                .map { executeCommand(it.body!!, this, userIsMod(it)) }
+                .map { executeCommand(it.body!!, this, userIsMod(it), it.author) }
 
             when {
                 results.any { it.isLeft() && (it as Either.Left).a is FailedModuleResponse } -> {
-                    addRawRestrictedComment("Command execution failed", "helper")
+                    addRawRestrictedComment("$staffAuthor Command execution failed", "helper")
                     results.first { (it as Either.Left).a is FailedModuleResponse }.bind()
                 }
                 results.any { it.isRight() } -> {
-                    addRawRestrictedComment("Command execution was successful", "helper")
+                    addRawRestrictedComment("$staffAuthor Command execution was successful", "helper")
```

Comment:
Maybe it would be good to include a colon (`:`) after the executor name to structure the message a little bit more.